src/en/help-oracle.md

```diff
@@ -0,0 +1,180 @@
+Title: Using the Oracle Compute Provider
+TODO: Test Oracle
+      Rewrite so its not so list heavy (especially Networks and spaces section)
+
+# Using the Oracle Compute Provider
+
+!!! Note:
+	Oracle Compute support is currently in [Juju 2.2 beta3][jujubeta] and
+        is experimental. There may be bugs. Work is continuing on improving the user
+        experience.
+
+To start using Juju with the [Oracle Compute Cloud][compute], some initial set up steps
+are required:
+
+1. Associate Ubuntu or Windows images from the marketplace with your Oracle account.
+   (this is an Oracle legal requirement).
+
+1. If not using a paid account, use `juju add-cloud` to set up the region and
+   endpoint details for Juju to use.  Set up credentials using `juju
+   add-credentials`. 
+
+1. Set up credentials using `juju add-credentials`.
+
+These steps - outlined below - will be refined as we work with Oracle to
+improve the user experience.
+
+Juju and Oracle can be used with either a free Oracle trial account, or a paid
+account.
+
+## Associate images
+
+For each Ubuntu series on which Juju is to run workloads, the relevant image
+needs to be selected from the Marketplace and associated with your account. You
+also need to agree to Oracle’s Terms and Conditions.
+
+The currently available Ubuntu images are:
+
+| Series   | Version                          |
+|----------|----------------------------------|
+| precise  | Ubuntu.12.04-LTS.amd64.20170417  |
```

Comment:
I left this as I'm uncertain as to whether this is how it appears in the Oracle dashboard.